### DISCUSSION TITLE
Can't import style css files

### DISCUSSION THREAD

**Discussion by iamdumonde** [primefaces/community#2398] [PrimeVue] [answered]
I have a problem importing css files from the resources folder located in node_modules/primevue.
As a result, I can't properly use the Editor component I need. 
I did all the installations, npm install primevue, but nothing, I tried to copy the resources folder from another project which I copied into the new one, but the import still didn't work, ... The problem is that after installation, the “resources” folder is not found... and the imports don't work: 

import 'primevue/resources/themes/saga-blue/theme.css';
import 'primevue/resources/primevue.min.css';
import 'primeicons/primeicons.css';

**Comment by RadithSandeepa** [comment, marked as answer]
It sounds like you’re having trouble with importing CSS files for PrimeVue components. Here's a step-by-step guide to help resolve the issue:

1.Verify Installation:

Ensure that you have installed PrimeVue and its dependencies correctly. Run the following commands in your project directory:

`npm install primevue primeicons`

2.Check the Package Structure:

Verify that the CSS files are indeed available in the node_modules directory. The path should be something like this:

- node_modules/primevue/resources/themes/saga-blue/theme.css
- node_modules/primevue/resources/primevue.min.css
- node_modules/primeicons/primeicons.css

3.Correct Import Paths:

Make sure your import paths in your JavaScript or TypeScript file are correct. Your imports should look like this:

- import 'primevue/resources/themes/saga-blue/theme.css';
- import 'primevue/resources/primevue.min.css';
- import 'primeicons/primeicons.css';


4.Restart Development Server:

Sometimes changes in node_modules require a restart of your development server. Stop your server and start it again:

`npm start`

5.Check webpack.config.js or vite.config.js:

If you are using a bundler like Webpack or Vite, make sure it is configured to handle CSS imports. For example, if you are using Webpack, ensure that the CSS loader is configured correctly.

6.Troubleshoot Path Issues:

If the files are not being found, you can also try using an absolute path to ensure that the correct files are being referenced:

- import 'node_modules/primevue/resources/themes/saga-blue/theme.css';
- import 'node_modules/primevue/resources/primevue.min.css';
- import 'node_modules/primeicons/primeicons.css';

7.Check for Installation Issues:

Sometimes issues arise from corrupted installations. Try deleting the node_modules folder and package-lock.json file, then reinstall:

```
rm -rf node_modules package-lock.json
npm install
```

8.Consult Documentation:

Refer to the PrimeVue documentation for any updates or changes regarding the usage and importing of CSS files.

If you've followed these steps and are still encountering issues, please provide more details about the error messages or behavior you’re experiencing.

**Reply by iamdumonde** [reply]
Thank you so much for your help @RadithSandeepa . 
I was able to read the documentation and realized that in the newly updated version the css files were no longer important, and Primevue says it has removed the resources folder. 
So I was able to readjust my code according to the standards described in the documentation. 
But I'm having another problem with Primevue's Editor component.

I'm working on an e-learning application, and I'm creating courses, with the Editor component I'm able to create the course well. 

I'm working with laravel vuejs, the problem is in the modification of the created course, you have to call the course and modify it, but in the Editor component, with v-model=“formData.text”, the text isn't displayed, whereas it's displayed in a textarea, or an input, that's where I've been stuck for a while, and with the doc I'm still not making any progress.

![image](https://github.com/user-attachments/assets/ca16fe39-a2e0-4ce5-b657-6471115908c6)
![image](https://github.com/user-attachments/assets/e4e6f04a-2513-48a1-9d6b-40bf86106ffb)

**Reply by RadithSandeepa** [reply]
I'm glad to hear you resolved the CSS import issue. Regarding the new problem with PrimeVue's Editor component, it sounds like a common issue related to how v-model is handled in custom components.

Please check this link for more details and potential solutions: [PrimeVue GitHub Issue #5606](https://github.com/primefaces/primevue/issues/5606)

Keep up the great work! Your persistence in tackling these issues is commendable, and I'm confident you'll find a solution soon. Don't hesitate to reach out if you need further assistance.

**Reply by sanimra** [reply]
Hello @iamdumonde, how did you solve your issue. I am facing the same problems. The css files are not supported and the v-model is not displaying in the editor
